exporter/clickhousemetricsexporterv2/internal/fingerprint.go

```diff
@@ -0,0 +1,73 @@
+package internal
+
+import "go.opentelemetry.io/collector/pdata/pcommon"
+
+type FingerprintType struct{ s string }
+
+func (t FingerprintType) String() string { return t.s }
+
+var (
+	ResourceFingerprintType FingerprintType = FingerprintType{s: "resource"}
+	ScopeFingerprintType    FingerprintType = FingerprintType{s: "scope"}
+	PointFingerprintType    FingerprintType = FingerprintType{s: "point"}
+)
+
+type Fingerprint struct {
+	attributes Attributes
+	hash       uint64
+	typ        FingerprintType
+}
+
+func NewFingerprint(typ FingerprintType, offset uint64, attrs pcommon.Map, extras map[string]string) *Fingerprint {
+	attributes := NewAttributesFromPcommonMap(attrs)
```

Comment:
```suggestion
	attributes := NewAttributesFromPcommonMap(attrs)
```

This is happening the list number of times now.